include/madrona/importer.hpp

```diff
@@ -48,25 +103,60 @@ struct SourceInstance {
     uint32_t objIDX;
 };
 
+struct EmbreeLoader {
+    struct Impl;
+
+    EmbreeLoader(Span<char> err_buf);
+    EmbreeLoader(EmbreeLoader &&) = default;
+    ~EmbreeLoader();
+
+    std::unique_ptr<Impl> impl_;
+
+    Optional<MeshBVH> load(const SourceObject &obj,
+            const DynArray<SourceMaterial> &materials);
+};
+
+struct SourceAssetInfo {
```

Comment:
Is this specifically for textures? Why is a level of abstraction *above* an object necessary? An Object is already composed of multiple meshes.

Reply:
discussed in slack